functions/menu.bash

```diff
@@ -206,7 +206,8 @@ show_main_menu() {
   elif [[ "$choice" == "50"* ]]; then
     choice2=$(whiptail --title "openHABian Configuration Tool — $(get_git_revision)" --menu "Backup/Restore" 14 116 7 --cancel-button Back --ok-button Execute \
     "50 | Backup openHAB config"      "Backup the current active openHAB configuration" \
-    "51 | Restore an openHAB config"  "Restore a previous openHAB configuration from backup" \
+    "51 | Restore an openHAB config"  "Restore an openHAB configuration from backup zipfile" \
+    "   | Restore text only config"   "Restore text only configuration without restarting" \
```

Comment:
Perhaps we should add a prompt to more fully explain what a text only restore really is for users as for a new user they may not understand what that means and it may not be what they want. Also does openhab support creating text only backups, and if so do we want to as well?

Reply:
ok I added a prompt.
It actually only extracts the text config files no matter if there's other files in the zipfile as well so backup creation is unaffected.